src/ontology/mondo-edit.obo

```diff
@@ -432964,6 +432964,16 @@ intersection_of: MONDO:0005550 ! infectious disease
 intersection_of: realized_in_response_to_stimulus NCBITaxon:1335626 ! Middle East respiratory syndrome-related coronavirus
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100125
+name: hallucinogen-persisting perception disorder
+def: "A perceptual disorder caused by intoxication with hallucinogen drugs, especially LSD. It is characterized by the recurrence of perceptive disturbances that first develop during intoxication. The contents of the perception and visual imagery range extensively and symptoms may include visual disturbances, hallucinations, and psychoses." [PMID:29547576]
+synonym: "flashbacks" EXACT [DSM-5:292.89(F16.983), ICD10:F16.183]
```

Comment:
I'm not sure this should be an exact synonym

It seems this term is used for any involuntary memory 
https://en.wikipedia.org/wiki/Flashback_(psychology)#cite_note-ball2006-3

Perhaps "LSD flashback"

But this seems to name the phenomena/phenotype, not the disorder.

Reply:
I agree, LSD flashback seems more like the phenomena. "Flashbacks" was a synonym in DSM-V and ICD10, but I can remove it.